bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/models/Carousel.java

```diff
@@ -0,0 +1,63 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2018 Adobe Systems Incorporated
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.wcm.core.components.sandbox.models;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.Nonnull;
+
+import org.apache.sling.api.resource.Resource;
+import org.osgi.annotation.versioning.ConsumerType;
+
+import com.adobe.cq.export.json.ComponentExporter;
+import com.adobe.cq.export.json.ContainerExporter;
+
+/**
+ * Defines the {@code Carousel} Sling Model used for the {@code /apps/core/wcm/sandbox/components/carousel} component.
+ *
+ * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
+ */
+@ConsumerType
+public interface Carousel extends ContainerExporter {
+
+    /**
+     * Returns the list of Carousel items
+     *
+     * @return List of {@link Resource}
+     */
+    default List<Resource> getItems() {
```

Comment:
I'm thinking this should be something more structured, perhaps a `List<ListItem>` so that we can have a meaningful title and description for each. We could still use the path for resource inclusion in `carousel.html`

Reply:
I added `ResourceListItemImpl` at https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/263/files#diff-bd50a5fc78fcfeeb1a727e2d45bb1620